packages/local/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "koishi-plugin-booru-local",
+  "description": "Image service by local collection for Koishi",
+  "version": "0.1.0",
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "files": [
+    "lib",
+    "dist"
+  ],
+  "author": {
+    "name": "Maiko Tan",
+    "email": "maiko.tan.coding@gmail.com"
+  },
```

Comment:
Change this to yourself.

You could consider listing @MaikoTan or @Kabuda-czh as the maintainer though.